packages/core/theme/src/components/accordion.ts

```diff
@@ -62,13 +62,14 @@ const accordionItem = tv({
       "group-[.is-splitted]:shadow-medium",
       "group-[.is-splitted]:rounded-medium",
     ],
-    heading: "",
+    heading: "relative",
     trigger: [
-      "flex py-4 w-full h-full gap-3 outline-none items-center tap-highlight-transparent",
+      "flex py-4 w-full h-full gap-3 outline-none items-center tap-highlight-transparent ",
```

Comment:
```suggestion
      "flex py-4 w-full h-full gap-3 outline-none items-center tap-highlight-transparent",
```

Unnecessary white space.